pkg/querier/frontend/frontend.go

```diff
@@ -38,6 +38,7 @@ var (
 	errTooManyRequest   = httpgrpc.Errorf(http.StatusTooManyRequests, "too many outstanding requests")
 	errCanceled         = httpgrpc.Errorf(StatusClientClosedRequest, context.Canceled.Error())
 	errDeadlineExceeded = httpgrpc.Errorf(http.StatusGatewayTimeout, context.DeadlineExceeded.Error())
+	maxBodySize         = 10 * 1024 * 1024 // 10 MiB
```

Comment:
Should be const.

Reply:
Remember to mention this as a `[CHANGE]` in the changelog. We basically don't accept anymore any query POST request with a body larger than 10MB.

Reply:
Added, let me know if you want it phrased differently